### PR TITLE
=clu #17253 Warn about too small cluster dispatcher

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #17253

File: akka-docs/rst/java/cluster-usage.rst [hunk 1]
```diff
@@ -704,7 +704,13 @@ For this purpose you can define a separate dispatcher to be used for the cluster
...
-      parallelism-max = 4
+      parallelism-min = 5
+      parallelism-max = 5
     }
   }
+
+.. warning:: 
+
+   Don't use a dispatcher with less than 5 threads, because otherwise there is a risk of 
+   deadlock when initializing the Cluster extension. This limitation does not exist in in Akka 2.4.
```
Comment (2beaucoup) [inline akka-docs/rst/java/cluster-usage.rst]: "in in" (2x)

Comment (patriknw) [inline akka-docs/rst/java/cluster-usage.rst]: okey, thanks

**Comment by drewhk** [comment]
Can't we backport the fix instead? Is it a risky change?

**Comment by patriknw** [comment]
We only backport critical fixes. There is an easy workaround, and everything works fine with default settings.

**Comment by drewhk** [comment]
Ok, got it.

**Comment by ktoso** [comment]
LGTM, please squash?

**Pull request merged**